modules/nf-core/iqtree/main.nf

```diff
@@ -8,20 +8,21 @@ process IQTREE {
         'biocontainers/iqtree:2.1.4_beta--hdcc8f71_0' }"
 
     input:
-    path alignment
+    tuple val(meta), path(alignment)
     val constant_sites
 
     output:
-    path "*.treefile",    emit: phylogeny
-    path "versions.yml" , emit: versions
+    tuple val(meta), path("*.treefile") , emit: phylogeny
```

Comment:
As we talked about, I could see users wanting to use the bootstrap tree output. Might be worth to add another channel for this and document that the channel is empty unless `bootstrap` is passed in the args (or even to make it a value input channel, not sure what's better).

Reply:
Added bootstrap output